### PR TITLE
Rollback sveltekit v2

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

ic-js bundled in a SvelteKit v2 app leads to issue on iOS <= v15.

# Changes

- Revert PR #4057

# Screenshots

image

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dfinity/nns-dapp",
-  "version": "2.0.61",
+  "version": "2.0.60",
```
Comment (dskloetd) [inline frontend/package.json:3]: This doesn't look right. How did you create the rollback?
When I do `git revert a1598bc03bcebdf8460fcda3ccb67d98a14fd6c9`, it doesn't affect the nns-dapp version like this.

File: frontend/src/tests/e2e/images.spec.ts [hunk 2]
```diff
@@ -45,7 +42,7 @@ test.describe.configure({ retries: 2 });
 
 test("Test images load on accounts page", async ({ page, context }) => {
   await page.goto("/accounts");
-  await expect(page).toHaveTitle("My ICP Tokens / NNS Dapp");
```
Comment (dskloetd) [inline frontend/src/tests/e2e/images.spec.ts:48]: This looks unrelated as well?

File: frontend/vitest.config.ts
Comment (dskloetd) [inline frontend/vitest.config.ts:1]: The changes to this file look much bigger than in the original PR.

**Review by dskloetd — COMMENTED** [review]
This doesn't look like a clean rollback?

**Comment by peterpeterparker** [comment]
Duplicated by #4149
